vm/method-verifier.cpp

```diff
@@ -531,11 +531,19 @@ MethodVerifier::verifyJumpOffset(cell_t offset)
 }
 
 bool
+MethodVerifier::verifyDimensionCount(cell_t ndims)
+{
+	if (ndims < 0 || ndims > sDIMEN_MAX) {
```

Comment:
This can be `ndims <= 0` I think, generating a 0-dimension array doesn't make sense?

This is good to merge w/ that and indentation/spacing fixed.